reactioncommerce/reaction
# refactor: demeteorize send welcome email / account creation

Resolves parts of #5529
Impact: minor
Type: feature|refactor

## Issue
Account creation, after the Meteor `user` has been created, is currently intertwined in a Meteor.startup function. This calls the meteorized `sendWelcomeEmail`, as well as using meteor for many account creation functions.

## Solution
- Create a `createAcccount` GraphQL mutation to decouple meteor from `Account` creation
- Demeteorize `sendWelcomeEmail` util

## Breaking changes
Any app with a custom account creation flow, which customizes `Accounts.createUser`, may need to update to the new GraphQL mutation.

## Testing
1. Register for an account
1. See you get the welcome email
1. See Account is created in both `Accounts` and `users` collections
